p/parquet-format/parquet_format_ubi_8.5.sh

```diff
@@ -18,7 +18,7 @@
 #
 # ----------------------------------------------------------------------------
 PACKAGE_NAME=parquet-format
-PACKAGE_VERSION=${1:-apache-parquet-format-2.9.0}
+PACKAGE_VERSION=${1:-apache-parquet-format-2.10.0}
```

Comment:
@vinodk99 update build_info and also version line in header of script to reflect latest version supported

Reply:
@nikhil-kalbande ,Updated the header with the latest version and updated the build_info.json file as suggested.